web/styles/app_components.css

```diff
@@ -1145,6 +1145,10 @@ div.overlay {
     }
 }
 
+.dropdown_widget_with_label_wrapper {
+    margin-top: 0 !important;
```

Comment:
We try to avoid overriding CSS properties using `important`. Maybe we can refactor the existing CSS which adds the 10px margin in some way. I guess the margin of 10px was added to have some space between the heading of first subsection and the setting label, so maybe we can refactor the CSS to do something like add a margin only on direct children `div` elements or `div` with a particular class.

Reply:
div with a particular class is recommended; direct children selectors as the last selector in a rule cause performance issues for the rest of the app and are being purged.